roles/openshift_logging/tasks/delete_logging.yaml

```diff
@@ -95,3 +95,36 @@
     - logging-elasticsearch
     - logging-fluentd
     - logging-mux
+
+# delete eventrouter
+- name: Delete EventRouter service account
+  oc_serviceaccount:
+    state: absent
+    name: "aggregated-logging-eventrouter"
+    namespace: "{{ openshift_logging_eventrouter_namespace | default('default') }}"
```

Comment:
I think we should be able to represent this in the defaults/main.yaml so we dont have to repeat everywhere

Reply:
We have the "per logging component" role defaults but one global uninstall task for logging. Those defaults are not present in this role. Once we refactor the uninstall task to be also per role, the "per logging component" roles defaults will be automatically picked up.

Otherwise, I am not sure how well will the variable precedence behave if we have same named variable defaulted at two different roles.

- https://github.com/wozniakjan/openshift-ansible/blob/b2ab2c642567be9400c2223c688046e812399f5b/roles/openshift_logging_eventrouter/defaults/main.yaml#L9

Reply:
This is partly why I suggested to move the delete tasks to the event router role.  Tasks can be called from other roles.  A case can be made that we may wish to devise deployment of something like eventrouter->kafka which would mean there is no need for any of the logging role.  Keeping it in logging would mean to uninstall we would have to rely upon the parent logging role.  I generally see event gathering not necessarily tied explicitly to logging

Reply:
Per my previous comment, moving all this to the role should alleviate your concerns